src/lib/i18n-data.js

```diff
@@ -1,21 +1,25 @@
 import en from "lib/i18n-data/en.json";
 import es from "lib/i18n-data/es.json";
 import fr from "lib/i18n-data/fr.json";
+import frfr from "lib/i18n-data/fr_fr.json";
 import ht from "lib/i18n-data/ht.json";
+import jajp from "lib/i18n-data/ja.json";
```

Comment:
I think I like the standard of using two letter variables until we have 2 of the same language. So ditto for Chinese